src/Tribe/Admin/Notices.php

```diff
@@ -270,4 +271,73 @@ public function maybe_display_unsupported_currency_notice() {
 			]
 		);
 	}
+
+	/**
+	 * Display notice for Paystack promotion.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	function maybe_display_paystack_notice() {
+
+		// Bail if we aren't in Tickets > Settings.
+		if ( \Tribe\Tickets\Admin\Settings::$settings_page_id !== tribe_get_request_var( 'page' ) ) {
+			return;
+		}
+
+		// Bail if Paystack plugin is installed and activated.
+		if ( class_exists( 'paystack\tec\classes\Core' ) ) {
```

Comment:
I may be incorrect, but I recall hearing that when we use `class_exists` we should set autoload to false. You may want to double check on this.

Reply:
You may be right, although this is for a plugin that isn't ours. It won't hurt to add it, though.